src/api/goods.js

```diff
@@ -3,9 +3,20 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll() {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error(`${response.status} - ${response.statusText}`);
+      }
+
+      return response.json();
+    });
 }
 
-export const get5First = () => {};
+export const get5First = () => getAll()
+  .then(goods => goods.sort(
+    (good1, good2) => good1.name.localeCompare(good2.name),
+  ))
+  .then(goods => goods.slice(0, 5));
```

Comment:
Chained array methods is better to keep at the same level.

```suggestion
  .then(goods => goods
    .sort((good1, good2) => good1.name.localeCompare(good2.name)))
    .then(goods => goods.slice(0, 5));
```